src/qoi/src/rayfire_mesh.C

```diff
@@ -1349,4 +1237,35 @@ namespace GRINS
 
   }
 
+  bool RayfireMesh::system_solve_3x3(libMesh::DenseMatrix<libMesh::Real> & A, libMesh::DenseVector<libMesh::Real> & b, libMesh::DenseVector<libMesh::Real> & x)
+  {
+    bool has_soln = false;
+    libMesh::Real det =   A(0,0) * (A(1,1)*A(2,2) - A(1,2)*A(2,1))
+                        - A(0,1) * (A(1,0)*A(2,2) - A(1,2)*A(2,0))
+                        + A(0,2) * (A(1,0)*A(2,1) - A(1,1)*A(2,0));
+
+    if (std::abs(det) >= 1.0e-15)
```

Comment:
Can you please make the tolerance `std::numeric_limits<libMesh::Real>::epsilon*10` or something similar? That way if libMesh::Real changes under-the-hood this also changes.

Reply:
I didn't even know that was a thing...

Sure, I'll update it and squash it in today